cypress.config.js

```diff
@@ -26,9 +26,16 @@ module.exports = defineConfig({
       grepFilterSpecs: true,
       grepOmitFiltered: true,
     },
-    setupNodeEvents(on, config) {
+    setupNodeEvents(cypressOn, config) {
+      const on = require("cypress-on-fix")(cypressOn);
+      const allSpecs = findCypressSpecs();
+      const chunk = getChunk(allSpecs, k, n);
+
       config = require("./cypress/support/cypress-grep/plugin")(config);
       config = require("./cypress/plugins/index.js")(on, config);
+      config = require("cypress-split")(on, config);
+      config.specPattern = chunk;
+
```

Comment:
It won't work. In "./cypress/support/cypress-grep/plugin" we are changing specPattern to run only specs with have tests with tags. Than you change this specPattern, so even if parallel will work, tags stop working. 
![image](https://github.com/saleor/saleor-dashboard/assets/15987043/14df5413-a3b1-419b-bdf8-8ed9a2b14d6c)
Can we just do that, without playing with spec pattern?